src/Refactoring-Core/RBReplaceCalledMethodByAnotherRefactoring.class.st

```diff
@@ -0,0 +1,72 @@
+"
+I'm a refactoring operation for replace one method call by another one.
+
+The another method name should have the same number of arguments.
+
+All senders of this method are changed by the other
+"
+Class {
+	#name : #RBReplaceCalledMethodByAnotherRefactoring,
```

Comment:
I think class names can be simplified. Simple RBReplaceMethodRefactoring would be good enough IMHO